test/spec/fixtures/schema.sql

```diff
@@ -1955,12 +1952,13 @@ create table whatev_sites (
 );
 
 create table whatev_jobs (
-  job_id        uuid    primary key
+  job_id        uuid
 , name text
 , site_id_1     int not null references whatev_sites (id)
 , project_id_1  int not null references whatev_projects (id)
 , site_id_2     int not null references whatev_sites (id)
 , project_id_2  int not null references whatev_projects (id)
+, primary key(job_id, site_id_1, project_id_1, site_id_2, project_id_2)
```

Comment:
This PK was added so this test keeps passing:

https://github.com/PostgREST/postgrest/blob/9bdb4e5c4f723c01f1a77f7b03575c7f8efb368f/test/spec/Feature/Query/EmbedDisambiguationSpec.hs#L121-L157

I do still wonder if this type of modeling can happen in the real world. Still, keeping this for completion.